src/ctx.cpp

```diff
@@ -779,8 +779,8 @@ void FunctionEmitContext::jumpIfAllLoopLanesAreDone(llvm::BasicBlock *target) {
         allDone = MasksAllEqual(finishedLanes, blockEntryMask);
     }
 
-    llvm::BasicBlock *bAll = CreateBasicBlock("all_continued_or_breaked");
-    llvm::BasicBlock *bNotAll = CreateBasicBlock("not_all_continued_or_breaked");
+    llvm::BasicBlock *bAll = CreateBasicBlock(llvm::Twine(llvm::Twine("all_continued_or_breaked")));
```

Comment:
Why `twine(twine(...))`?

Reply:
Do we need to construct `Twine` explicitly? I think if the function accepts `Twine` as an argument, then it will be constructed if a matching `Twine` constructor is found. I.e. no need to change invocation.

Reply:
Ah. I cleaned it up.